src/Microsoft.DncEng.SecretManager/SecretPropertiesWrapper.cs

```diff
@@ -0,0 +1,13 @@
+namespace Microsoft.DncEng.SecretManager;
```

Comment:
You can use [SecretModelFactory](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.Security.KeyVault.Secrets/4.4.0/api/Azure.Security.KeyVault.Secrets/Azure.Security.KeyVault.Secrets.SecretModelFactory.html) to create `SecretProperty` instances. I think that will be enough to eliminate this extra wrapper class.

Reply:
Oh, I see you're also tracking NextRotationFound. Hmm...